src/main/resources/schema.sql

```diff
@@ -51,6 +51,29 @@ create table if not exists academy_project.diagram
 ALTER TABLE academy_project.diagram
     OWNER TO CURRENT_USER;
 
+create table if not exists academy_project.metadata
+(
+    id           uuid primary key not null,
+    name         text             not null,
+    size         bigint           not null,
+    crc          bigint,
```

Comment:
I know the crc, as well as other variables such as download_uri, have to be not null, but in order to test only the back-end functionality I needed them to be null for the time being, but I will make sure to edit the table once the front-end is up and running.